### ISSUE TITLE
Burying timer mines

### ISSUE THREAD

**Issue by DSx61x4** [PlayersCommittee/gemp-swccg-public#231]
Currently no option exists to bury timer mines per the rules.

**Comment by billbisco** [comment]
Yes this feature is missing and needs to be enabled.  Thank you.  

Burying Mines
If your mining droid is present at an exterior planet site,
it may 'bury' (place face down) mines there to simulate
creation of a minefield. You may bury any number of
cards from your hand face down underneath that site,
but only during your deploy phase. You may choose to
bury 'real' mines, 'duds' (non-mine cards buried to try
and trick your opponent) or a mixture of the two. When
any character, vehicle or starship deploys or moves to
or across that site, all buried cards there are 'tripped'
(revealed). Any duds are simply lost. Any mines
'explode,' targeting the card that tripped them if
applicable. These mines target as normal automated
weapons (see Automated Weapons rules). If buried
mines are tripped during your turn and you have a
mining droid present, you may choose to 'defuse' any or
all of them (at normal use of the Force) before they
explode.

**Comment by billbisco** [comment]
Note that a similar feature has been implemented for Tatooine Bluffs and the Watto Objective from Tatooine set.  So some coding should be able to be re-used and make implementation not too hard.

**Comment by billbisco** [comment]
This is a missing Advanced Rulebook capability, not a broken card.

AR: during your deploy phase, if your mining droid is present at an exterior planet site, you may bury any number of cards from hand face down under that site (real mines, duds, or both). When a character, vehicle, or starship deploys or moves to or across that site, buried cards are tripped. Duds are lost. Mines explode as automated weapons. If that happens on your turn and you have a mining droid present, you may defuse before they explode.

Timer Mine / Infantry Mine / Vehicle Mine currently only deploy as face-up automated weapons. There is no bury / trip / defuse action. That is the gap.

This is a minefield engine, not a leftover card-text patch. Leave the issue open as that feature. No code in this pass.
